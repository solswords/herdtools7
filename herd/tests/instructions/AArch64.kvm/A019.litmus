AArch64 A019
TTHM=P0:HD
Variant=vmsa
{
 0:X1=x; 0:X2=7; 0:X0=42;
 TTD(x)=(oa:PA(x),db:0,dbm:1);
}
P0             ;
CAS W0,W2,[X1] ;
exists TTD(x)=(oa:PA(x),db:0,dbm:1)
