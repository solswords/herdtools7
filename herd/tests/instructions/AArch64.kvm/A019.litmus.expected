Test A019 Allowed
States 2
[PTE(x)]=(oa:PA(x), db:0, dbm:1);
[PTE(x)]=(oa:PA(x), dbm:1);
Ok
Witnesses
Positive: 2 Negative: 3
Condition exists ([PTE(x)]=(oa:PA(x), db:0, dbm:1))
Observation A019 Sometimes 2 3
Hash=f45eee29aa2a126c7370f37a156463ea

